src/Api/Api.ts

```diff
@@ -1,13 +1,9 @@
-import { Method, HeadersMap, ApiResponse } from './types';
-import createRequest from './createRequest';
+import { HeadersMap } from '../types';
 
-const withAccessToken = (headers: HeadersMap = {}, accessToken: string): HeadersMap => ({
-    ...headers,
-    'x-access-token': accessToken,
```

Comment:
Decided not to couple the `accessToken` with `Api`. The `BaseSdk` now takes care of that, along with the `User-Agent` header.

Reply:
May I ask why? 

AFAIK all Clean Code 101 articles are stating that composition should always be preferred over inheritance. If possible.

In my arrogant opinion 😜, headers and base URI are http concepts and it totally makes sense to couple them to http client instance.

Reply:
> AFAIK all Clean Code 101 articles are stating that composition should always be preferred over inheritance. If possible.

I don't know what does composition/inheritance have to do with it...

> totally makes sense to couple them to http client instance.

But I agree :+1:

Reply:
> composition should always be preferred over inheritance

I  do not follow. What inheritance? 
Why should `Api` module know about our specific implementation of `accessToken` header (which is not following the usual [`Authorization` header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Authorization) pattern)? 
My idea of `Api` is to one day extract it and use as a dependency to our projects instead of copying it around. It should be composable enough that anyone else can use it in their code. For this reason, I don't want it to be opinionated of the `accessToken`. This change achieves my goal. It should be able to perform requests to *any* endpoint, not only to the Prezly API. 

On the other hand, the `Sdk` module is tied to our API, so it should be the part of the code responsible to provide correct authorization to the API. For the same reason, I've decided to place `User-Agent` header inside `Sdk` module.

Reply:
Me, Marko and Kamil had a call about this and resolved the confusion around this thread.

```js
const apiClient = new ApiClient({
    headers: {
        'X-Access-Token': token,
        'User-Agent': userAgent,
    },
    baseUri,
});

const coverage = new Coverage(apiClient);
```

Reply:
Changed with https://github.com/prezly/javascript-sdk/pull/8/commits/e439047640cf24dd1c2216c0212ad26284d7594d.